python/redmonster/datamgr/io2.py

```diff
@@ -253,7 +255,9 @@ def __init__(self, zpick, dest=None, clobber=True):
             run2d = environ['RUN2D']
             run1d = environ['RUN1D']
             if bsr and run2d and run1d:
-                testpath = join(bsr, run2d, '%s' % zpick.plate, run1d)
+                testpath = join(bsr, run2d, '%s'%
```

Comment:
Related to my comment about the version above, this makes the presence or absence of the 'v' a bit more complicated, since this is supposed to change *e.g.* v1.2.0 into v1_2_0, which is a BOSS-like version string.